provider/bigquery.go

```diff
@@ -1301,8 +1301,12 @@ func (store *bqOfflineStore) GetTrainingSet(id ResourceID) (TrainingSetIterator,
 	return store.newbqTrainingSetIterator(iter), nil
 }
 
-func (store *bqOfflineStore) GetTrainingSetTestSplit(id ResourceID, testSize float32, shuffle bool, randomState int) (TrainingSetIterator, TrainingSetIterator, func() error, error) {
-	return nil, nil, nil, fmt.Errorf("not Implemented")
+func (store *bqOfflineStore) CreateTrainTestSplit(def TrainTestSplitDef) (func() error, error) {
```

Comment:
swap this to `NewInternalError` or similar and add a bit of detail, kind of like:

`provider health check not implemented`